packages/pluggableWidgets/column-chart-native/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "column-chart-native",
+  "widgetName": "ColumnChart",
+  "version": "1.0.0",
+  "license": "Apache-2.0",
+  "copyright": "© Mendix Technology BV 2022. All rights reserved.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mendix/widgets-resources.git"
+  },
+  "config": {},
+  "packagePath": "com.mendix.widget.native",
+  "scripts": {
+    "start": "pluggable-widgets-tools start:native",
+    "build": "pluggable-widgets-tools build:native",
+    "format": "pluggable-widgets-tools format",
+    "release": "pluggable-widgets-tools release:native",
+    "lint": "eslint --config ../../../.eslintrc.js --ext .jsx,.js,.ts,.tsx src/",
+    "test:e2e:local:android": "detox test --configuration android.emulator.developerapp",
+    "test:e2e:local:ios": "detox test --configuration ios.simulator.developerapp",
+    "debug:e2e:local:android": "detox test --record-logs all -l trace --configuration android.emulator.developerapp",
+    "debug:e2e:local:ios": "detox test --record-logs all -l trace --configuration ios.simulator.developerapp",
+    "test": "pluggable-widgets-tools test:unit:native",
+    "version": "npm run release"
+  },
+  "dependencies": {
+    "@mendix/piw-utils-internal": "^1.0.0",
+    "victory-native": "^35.5.5"
+  },
+  "devDependencies": {
+    "@mendix/pluggable-widgets-tools": ">=8.9.2",
+    "detox": "^19.5.7",
+    "eslint": "^7.20.0",
+    "victory-bar": "35.11.4"
```

Comment:
You don't need to add `victory-bar` here.

Reply:
done